src/rustllvm/PassWrapper.cpp

```diff
@@ -97,6 +97,55 @@ LLVMRustAddPass(LLVMPassManagerRef PM, Pass *pass) {
     pm->add(pass);
 }
 
+#define GEN_SUBTARGETS \
+    SUBTARGET(X86) \
+    SUBTARGET(ARM) \
+    SUBTARGET(AArch64) \
+    SUBTARGET(Mips) \
+    SUBTARGET(PPC)
```

Comment:
Support for each subtarget is conditional most of the time because not all LLVMs are built with all these targets enabled, so this'll need to take that into account somehow